source/includes/_payouts.md

```diff
@@ -30,15 +30,15 @@ account.
 | Attribute          | Description                                                                       |
 | --------------------------- | --------------------------------------------------------------------------------- |
 | `id`                        | Unique identifier for the object.                                                                                                                                 |
-| `amount`                    | Decimal formatted string of the gross amount.                                                                                                                     |
-| `cost`                      | Decimal formatted string of the salary cost.                                                                                                                      |
+| `amount`                    | Decimal formatted string of the gross salary amount.                                                                                                                     |
+| `invoiced_amount`           | Decimal formatted string of the invoiced amount (gross salary+payroll tax+user fee).                                |
```

Comment:
This is only correct in Sweden. I agree that we should have describe the difference though. I'll write something.